File: 6-kyu/Find The Odd Int/FindOddInt.java
//Kata located at: https://www.codewars.com/kata/54da5a58ea159efa38000836

public class FindOdd {
	public static int findIt(int[] A) {
  	int n = 0;
    for(int num : A)  n ^= num;
    return n;
  }
}

File: 6-kyu/Find The Odd Int/FindOddInt.js
// Kata can be found at: https://www.codewars.com/kata/54da5a58ea159efa38000836

function findOdd(A) {
  var n = 0;
  for(var i = 0; i < A.length; i++)  n ^= A[i];
  return n;
}
